rest/class-boldgrid-backup-rest-test.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * File: class-boldgrid-backup-rest-test.php
+ *
+ * @link       https://www.boldgrid.com
+ * @since      X.X.X
+ *
+ * @package    Boldgrid_Backup
+ * @copyright  BoldGrid
+ * @version    $Id$
+ * @author     BoldGrid <support@boldgrid.com>
+ */
+
+/**
+ * Class: Boldgrid_Backup_Rest_Test
+ *
+ * REST endpoints to access the backup preflight check results.
+ *
+ * @since X.X.X
+ */
+class Boldgrid_Backup_Rest_Test extends Boldgrid_Backup_Rest_Controller {
+
+	/**
+	 * Resource name.
+	 *
+	 * @since  X.X.X
+	 * @access private
+	 * @var    string
+	 */
+	protected $resource = 'test';
+
+	/**
+	 * Register the routes for the objects of the controller.
+	 *
+	 * @since X.X.X
+	 */
+	public function register_routes() {
+		$this->register_get();
+	}
+
+	/**
+	 * Register the route for getting test results.
+	 *
+	 * @since X.X.X
+	 */
+	public function register_get() {
+		register_rest_route( $this->namespace, '/' . $this->resource, [
```

Comment:
Arrays should be declared with long array syntax per WordPress coding standards
https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#declaring-arrays

Reply:
See similar comment above.